packages/botonic-react/src/webchat/header.jsx

```diff
@@ -42,16 +54,24 @@ export const DefaultHeader = props => (
       <HeaderTitle>Botonic</HeaderTitle>
       <Subtitle>Online</Subtitle>
     </Flex>
+    <CrossHeader onClick={props.onChange}>X</CrossHeader>}
   </Diffuse>
 )
 
 export const WebchatHeader = props => {
   const { webchatState } = useContext(WebchatContext)
-
+  const handleCloseWebchat = event => {
+    props.onChange(event.target.value)
```

Comment:
I think it's not needed to pass argument in this function.

Reply:
you are right!

Reply:
> I think it's not needed to pass argument in this function.

@guinii https://developer.mozilla.org/en-US/docs/Web/API/Window/event it's deprecated to use the global event.